xml/admin_rbd.xml

```diff
@@ -1872,4 +1872,289 @@ protocol feature mismatch, my 2fffffffffff &lt; peer 4010ff8ffacffff missing 401
    </step>
   </procedure>
  </sect1>
+ <sect1 xml:id="rbd-kubernetes">
+   <title>Block Devices and &kube;</title>
+   <para>
+     You can use &ceph; RBD with &kube;v1.13 and higher through the <literal>ceph-csi</literal>
+     driver. This driver dynamically provisions RBD images to back &kube;
+     volumes and maps these RBD images as block devices (optionally mounting a
+     file system contained within the image) on worker nodes running pods that
+     reference an RBD-backed volume.
+   </para>
+   <para>
+     To use &ceph; block devices with &kube;, you must install and configure
+     <literal>ceph-csi</literal> within your &kube; environment.
+   </para>
+   <important>
+     <para>
+        <literal>ceph-csi</literal> uses the RBD kernel modules by default
+        which may not support all &ceph; CRUSH tunables or RBD image features.
+     </para>
+   </important>
+   <procedure>
+     <step>
+       <para>
+         By default, &ceph; block devices use the RBD pool. Create a pool for
+         &kube; volume storage. Ensure your &ceph; cluster is running, then
+         create the pool:
+       </para>
+<screen>&prompt.cephuser;ceph osd pool create kubernetes</screen>
+     </step>
+     <step>
+       <para>
+         Use the RBD tool to initialize the pool:
+       </para>
+<screen>&prompt.cephuser;rbd pool init kubernetes</screen>
+     </step>
+     <step>
+       <para>
+         Create a new user for &kube; and <literal>ceph-csi</literal>. Execute the following
+         and record the generated key:
+       </para>
+<screen>&prompt.cephuser;ceph auth get-or-create client.kubernetes mon 'profile rbd' osd 'profile rbd pool=kubernetes' mgr 'profile rbd pool=kubernetes'
+[client.kubernetes]
+    key = AQD9o0Fd6hQRChAAt7fMaSZXduT3NWEqylNpmg==</screen>
+     </step>
+     <step>
+       <para>
+         <literal>ceph-csi</literal> requires a ConfigMap object stored in &kube;
+         to define the the &ceph; monitor addresses for the &ceph; cluster.
+         Collect both the &ceph; cluster unique fsid and the monitor addresses:
+       </para>
+<screen>&prompt.cephuser;ceph mon dump
+&lt;...&gt;
+fsid b9127830-b0cc-4e34-aa47-9d1a2e9949a8
+&lt;...&gt;
+0: [v2:192.168.1.1:3300/0,v1:192.168.1.1:6789/0] mon.a
+1: [v2:192.168.1.2:3300/0,v1:192.168.1.2:6789/0] mon.b
+2: [v2:192.168.1.3:3300/0,v1:192.168.1.3:6789/0] mon.c</screen>
+     </step>
+     <step>
+       <para>
+         Generate a <filename>csi-config-map.yaml</filename> file similar to
+         the example below, substituting the fsid for <literal>clusterID</literal>, and
```

Comment:
```suggestion
         the example below, substituting the FSID for <literal>clusterID</literal>, and
```

Reply:
I am not comfortable with this unpronounceable (or at least hard-to-pronounce) initialism in lower-case, which is why I am capitalizing it. However, if this "fsid" term is standard usage then by all means ignore these suggestions.